From 090877ff855b914f49d4462e0384a83c4059390b Mon Sep 17 00:00:00 2001
From: Avra Sengupta <asengupt@redhat.com>
Date: Fri, 11 Mar 2016 14:01:31 +0530
Subject: [PATCH 29/80] snapshot/cli: Keep the dict keys uniform

snapshot info was using a different dictionary key
as compared to other snapshot commands. This was
throwing a recurring error log in cli.log

Change-Id: I9eb9a4541c10a45ec7673a23c89c85a7dce9f3ec
BUG: 1305456
Signed-off-by: Avra Sengupta <asengupt@redhat.com>
Reviewed-on: http://review.gluster.org/13677
Smoke: Gluster Build System <jenkins@build.gluster.com>
NetBSD-regression: NetBSD Build System <jenkins@build.gluster.org>
CentOS-regression: Gluster Build System <jenkins@build.gluster.com>
Reviewed-by: Rajesh Joseph <rjoseph@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/70551
Tested-by: Rajesh Joseph <rjoseph@redhat.com>
---
 cli/src/cli-cmd-parser.c                      |    2 +-
 xlators/mgmt/glusterd/src/glusterd-snapshot.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/cli/src/cli-cmd-parser.c b/cli/src/cli-cmd-parser.c
index 04c7ae2..1a4c2f5 100644
--- a/cli/src/cli-cmd-parser.c
+++ b/cli/src/cli-cmd-parser.c
@@ -4192,7 +4192,7 @@ cli_snap_info_parse (dict_t *dict, const char **words, int wordcount)
         cmd = GF_SNAP_INFO_TYPE_VOL;
 out:
         if (ret == 0) {
-                ret = dict_set_int32 (dict, "cmd", cmd);
+                ret = dict_set_int32 (dict, "sub-cmd", cmd);
                 if (ret) {
                         gf_log ("cli", GF_LOG_ERROR, "Could not save "
                                 "type of snapshot info");
diff --git a/xlators/mgmt/glusterd/src/glusterd-snapshot.c b/xlators/mgmt/glusterd/src/glusterd-snapshot.c
index 3365e5f..4912f56 100644
--- a/xlators/mgmt/glusterd/src/glusterd-snapshot.c
+++ b/xlators/mgmt/glusterd/src/glusterd-snapshot.c
@@ -3688,7 +3688,7 @@ glusterd_handle_snapshot_info (rpcsvc_request_t *req, glusterd_op_t op,
         GF_VALIDATE_OR_GOTO (this->name, dict, out);
 
 
-        ret = dict_get_int32 (dict, "cmd", &cmd);
+        ret = dict_get_int32 (dict, "sub-cmd", &cmd);
         if (ret) {
                 gf_msg (this->name, GF_LOG_ERROR, 0,
                         GD_MSG_DICT_GET_FAILED, "Failed to get type "
-- 
1.7.1

